sapl-spring-data-mongo-reactive/src/main/java/io/sapl/springdatamongoreactive/sapl/queries/enforcement/ReflectedMongoQueryCreator.java

```diff
@@ -32,7 +32,8 @@ protected ReflectedMongoQueryCreator() {
         this.mongoQueryCreator = Class.forName(MONGO_QUERY_CREATOR_NAME);
     }
 
-    protected Constructor<?> getDeclaredConstructor(Class<?> class1, Class<?> class2, Class<?> class3, Class<?> class4)
+    @SuppressWarnings("rawtypes")
```

Comment:
@SupressWarnings still not justified in general

Reply:
Yes I know, this should not be the last PR

Reply:
How do I justify a SupressWarning? What is meant by justifying a warning?